packages/block-library/src/image/style.scss

```diff
@@ -9,6 +9,7 @@
 		max-width: 100%;
 		vertical-align: bottom;
 		box-sizing: border-box;
+		width: fit-content;
```

Comment:
I can reproduce on Firefox. Here's what I'm seeing in the frontend.

<img width="430" alt="Screenshot 2024-10-21 at 11 35 30 am" src="https://github.com/user-attachments/assets/9d0cb741-093c-4701-91b5-de10c9de48cc">

This fix works for me, and doesn't appear to affect other stand alone images, or images in galleries as far as I can tell.

However, I noticed that images in the editor receive a height and width of `100%` when aspect ratio is enabled:

https://github.com/WordPress/gutenberg/blob/413bf9bf423fba3e130f6f08551f0030d499e76d/packages/block-library/src/image/edit.js#L422-L422

Adding the same to the frontend also addresses the issue.

So I'm wondering whether the editor and frontend CSS needs to be the same just to ensure parity?

Reply:
Setting dimensions should also fix it, and I believe parity is important. However, I think that's somehow something that we'd consider changing as it may be unexpected behavior. 

As a site admin, I may want to use aspect ratio for images without them taking the entire width of their container. The aspect ratio shouldn't be opinionated on the image dimension, and that should be treated separately.

Reply:
The editor has an extra wrapper div for the resizable box which requires min/max width/height props to constrain the image. That's why things are set to 100% on the image. As an aside, it causes some issues with images smaller than the container being too big in the editor, but that's a problem with the resizable box container, not the image.

Anyway, parity already broken here given the need for the resizable box container, so I think this solution is fine.

Reply:
Thanks for the explainer. LGTM